src/jit/gentree.h

```diff
@@ -4635,10 +4635,22 @@ struct GenTreePutArgStk : public GenTreeUnOp
     }
```

Comment:
Instead on encoding this in the upper 16 bits 
It seems better to me to add a new field on line 4549:

```
#if defined(UNIX_X86_ABI)
    unsigned gtPadAlign; // num of padding slots for stack alignment
#endif
```